platform/viewer/cypress/integration/common/OHIFCornerstoneToolbar.spec.js

```diff
@@ -6,22 +6,8 @@ describe('OHIF Cornerstone Toolbar', () => {
     });
 
     beforeEach(() => {
-      //Creating aliases for Cornerstone tools buttons
-      cy.get('.toolbar-button:nth-child(2)').as('stackScrollBtn');
-      cy.get('.toolbar-button:nth-child(3)').as('zoomBtn');
-      cy.get('.toolbar-button:nth-child(4)').as('levelsBtn');
-      cy.get('.toolbar-button:nth-child(5)').as('panBtn');
-      cy.get('.toolbar-button:nth-child(6)').as('lengthBtn');
-      cy.get('.toolbar-button:nth-child(7)').as('annotateBtn');
-      cy.get('.toolbar-button:nth-child(8)').as('angleBtn');
-      cy.get('.toolbar-button:nth-child(9)').as('resetBtn');
-      cy.get('.toolbar-button:nth-child(10)').as('cineBtn');
-      cy.get('.expandableToolMenu').as('moreBtn');
-      cy.get('.PluginSwitch > .toolbar-button').as('twodmprBtn');
-      cy.get('.btn-group > .toolbar-button').as('layoutBtn');
-      cy.get('.pull-right > .RoundedButtonGroup > .roundedButtonWrapper > .roundedButton').as('measurementsBtn');
-      cy.get('.viewport-element').as('viewport');
-      cy.get('section.sidepanel.from-right').as('measurementsPanel')
+      cy.initCornerstoneToolsAliases();
+      cy.initCommonElementsAliases();
```

Comment:
That's interesting. So these persist if set in `beforeEach`, but not if done in a separate test case?